src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+html {
+  background-color: antiquewhite;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  background-color: #ffff;
+  width: 100%;
+  height: 60px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  align-items: center;
+  text-transform: uppercase;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  justify-content: center;
+}
+
+.nav {
+  display: flex;
+  height: 100%;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  text-decoration: none;
+  color: #000;
+  margin: 0 10px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.is-active {
```

Comment:
This is not necessary, because `is-active` is assigned to an element that already has the class `nav__link`. And `nav__link` has `position: relative`